soc/silabs/silabs_s2/xg28/Kconfig.soc

```diff
@@ -1,24 +1,37 @@
 # Copyright (c) 2025 Christoph Jans
+# Copyright (c) 2025 Shontal Biton
 # SPDX-License-Identifier: Apache-2.0
 
-config SOC_SILABS_XG28
+config SOC_SERIES_EFM32PG28
 	bool
-	select SOC_FAMILY_SILABS_S2
+	select SOC_SILABS_XG28
 	help
-	  Silicon Labs XG28 Series SoC and modules
+	  Silicon Labs EFM32PG28 Series MCU
 
-config SOC_SERIES_EFM32PG28
+config SOC_SERIES_EFR32ZG28
 	bool
 	select SOC_SILABS_XG28
 	help
-	  Silicon Labs EFM32PG28 Series MCU
+	  Silicon Labs EFR32ZG28 Series MCU
+
+config SOC_SILABS_XG28
```

Comment:
Why did you move this symbol? Following the principle of declaration before use, it should stay at the top of the file. By moving this, you are also introducing an unnecessary diff that makes it harder to review the PR.